### PR TITLE
Do not require config.yml

### PR DESCRIPTION
**Pull request by mtojek**
Changes:
* do not require config.yml to be present
* update Dockerfile
* remove config.yml from the repository

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -37,7 +39,7 @@ var (
 	configPath       = "config.yml"
 
 	defaultConfig = Config{
-		PublicDir:           "config.yml",
+		PublicDir:           "public",
```
Comment (ruflin) [inline main.go:42]: ups :-(

File: main.go [hunk 2]
```diff
@@ -114,16 +116,18 @@ func main() {
 
 func getConfig() (*Config, error) {
 	cfg, err := ucfgYAML.NewConfigWithFile(configPath)
+	if os.IsNotExist(err) {
+		return &defaultConfig, nil
```
Comment (ruflin) [inline main.go:120]: Could you add a log message here that the default configs are used as no config was found?

File: config.yml [hunk 3]
```diff
@@ -1,5 +0,0 @@
-public_dir: "./public"
```
Comment (ruflin) [inline config.yml:1]: I would still like to keep this one to show users what configs are there. It is also part documentation.

Comment (mtojek) [reply config.yml:1]: How about `config.reference.yml`?

Comment (ruflin) [reply config.yml:1]: ++. Should we fall back to it or just have it there?

Comment (mtojek) [reply config.yml:1]: There won't be any `config.yml` by default as it's not required, but there is will be a `config.reference.yml` if users would like to preview the configuration file.

Comment (ruflin) [reply config.yml:1]: SGTM

File: main.go (hunk 2 above)
Comment (mtojek) [reply main.go:121]: Fixed

File: config.yml (hunk 3 above)
Comment (mtojek) [reply config.yml]: Fixed

**Pull request merged**